department-of-veterans-affairs/va.gov-cms
# VACMS-8016 claro recurring events fixes

## Description

Relates to #8016. Fixes the merge conflicts caused by my testing updates, fixes the button behavior inside the modal when editing a recurring event series.

PR is targeting the `integration-recrurring-events` branch.

## Testing done

- log in as admin
- create an event node with a recurring event. save the node.
- edit the saved node, edit the event series
- cancel any event
- hover over the buttons and see that the behavior is fixed

## Screenshots
<img width="816" alt="Screen Shot 2022-02-14 at 4 03 00 PM" src="https://user-images.githubusercontent.com/11279744/153967959-5614be4e-d735-4721-9830-5a12391f4ba3.png">


## QA steps
- log in as admin
- create an event node with a recurring event. save the node.
- edit the saved node, edit the event series
- cancel any event
- hover over the buttons and see that the behavior is fixed

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
